src/com/intellij/idea/plugin/hybris/runConfigurations/HybrisJUnitExtension.java

```diff
@@ -26,57 +28,80 @@
 import com.intellij.execution.configurations.RunnerSettings;
 import com.intellij.execution.junit.JUnitConfiguration;
 import com.intellij.idea.plugin.hybris.project.utils.HybrisRootUtil;
+import com.intellij.idea.plugin.hybris.properties.PropertiesService;
 import com.intellij.idea.plugin.hybris.settings.HybrisProjectSettings;
 import com.intellij.idea.plugin.hybris.settings.HybrisProjectSettingsComponent;
 import com.intellij.openapi.project.Project;
 import com.intellij.openapi.util.io.FileUtil;
 import com.intellij.openapi.vfs.VirtualFile;
+import java.util.StringTokenizer;
 import org.jetbrains.annotations.NotNull;
 
-import static com.intellij.idea.plugin.hybris.common.HybrisConstants.HYBRIS_DATA_DIRECTORY;
-import static com.intellij.idea.plugin.hybris.common.HybrisConstants.HYBRIS_DATA_DIR_ENV;
-
 public class HybrisJUnitExtension extends RunConfigurationExtension {
 
-    @Override
-    public <T extends RunConfigurationBase<?>> void updateJavaParameters(
-        final T configuration, final JavaParameters params, final RunnerSettings runnerSettings
-    ) throws ExecutionException {
-        if (runnerSettings != null || !isApplicableFor(configuration)) {
-            return;
-        }
-        final Project project = configuration.getProject();
-        final ParametersList vmParameters = params.getVMParametersList();
+  @Override
+  public <T extends RunConfigurationBase<?>> void updateJavaParameters(
```

Comment:
quite strange reformatting, 2 spaces vs 4 spaces, looks like not needed

Reply:
Yeah I Agree, I will revert the style changes.
Seems my Idea used a formetter from another project

Reply:
@StefanKruk , looks like this one is not yet reformatted, it can be GitHub diff, but can you please check?

Reply:
Checked, Fixed the Formatting.